mkfile

```diff
@@ -0,0 +1,6 @@
+present:V:
+	present .
+
+deps:V:
+	go install golang.org/x/tools/cmd/present
```

Comment:
maybe makes sense `go get -v golang.org/x/tools/cmd/present` in deps =s?

Reply:
very nice this tool, did not know before

Reply:
indeed it is =D

I'm always in doubt if go install will also already go get the code or if it just install if the code has already been got =P. Just running the go install failed on your host ?

Reply:
I have, just had to prefix the go get before, but I do not know, sometimes it was a bad configuration, the bin of the path was not configured correctly too

Reply:
I have, just had to prefix the go get before, but I do not know, sometimes it was a bad configuration, the bin of the path was not configured correctly too

Reply:
I have, just had to prefix the go get before, but I do not know, sometimes it was a bad configuration, the bin of the path was not configured correctly too

Reply:
I have, just had to prefix the go get before, but I do not know, sometimes it was a bad configuration, the bin of the path was not configured correctly too